community/releases/next-release-preview/index.md

```diff
@@ -12,3 +12,8 @@ in the next release of Spinnaker. These notes will be prepended to the release
 changelog.
 
 ## Coming Soon in Release 1.26
+
+### Clouddriver account sharding for caching improvements
+
+The sql backed clouddriver caching pods are now capable of sharding the accounts of any provider 
+by setting a new configuration property `caching.sharding-enabled` to `true`. A hashing logic is applied to determine which pod can run the caching agents of an account.
```

Comment:
I'd add that this feature is opt-in and disabled by default, it only works when using specifically the SQL backend scheduler (as you can have cache stored in SQL but still use the redis scheduler), and it works for all cloud providers by splitting accounts based on their name, so all caching agents for the same account are executed in the same pod, but not all pods will cache all accounts.

Reply:
Will update accordingly.